CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -11,18 +11,20 @@ import UIKit
 class EditDemographicsViewController: UIViewController {
 
     // MARK: - Private Data Vars
-    // TODO: Retrieve values from UserDefaults after saving from backend
-    private let name = "Lucy Xu"
-    private let year = "2021"
-    private let major = "Computer Science"
-    private let hometown = "Boston, MA"
-    private let pronouns = "She/Her/Hers"
-    // TODO: Update with networking values from backend
     private var classSearchFields: [String] = []
     private var fieldsEntered: [Bool] = [true, true, true, true, true] // Keep track of fields that have been entered
-    private let hometownSearchFields = ["Boston, MA", "New York, NY", "Washington, DC", "Sacramento, CA", "Ithaca, NY"]
+    private let fieldMap = [
+        Constants.UserDefaults.userFullName,
+        Constants.UserDefaults.userGraduationYear,
+        Constants.UserDefaults.userMajor,
+        Constants.UserDefaults.userHometown,
+        Constants.UserDefaults.userPronouns
+    ]
+    private var fieldValues: [String: String] = [:] // Keep track of selected values
```

Comment:
`fieldMap` and `fieldValues` use a lot of string literals and indices to handle the information, which makes it a little tough to understand what's going on and might make it more error prone if something doesn't line up

Might be better to make a struct class that handles the information (called `Demographics` or something) instead of a dict and list, so that way it's all type safe

 In `savePressed` you would do something like
```
if let major = demographics.major, ...
```
over
```
if let graduationYear = fieldValues[fieldMap[1]]
```